doc/api/stream.md

```diff
@@ -1378,6 +1381,8 @@ constructor and implement the `writable._write()` method. The
     it becomes possible to write JavaScript values other than string,
     `Buffer` or `Uint8Array` if supported by the stream implementation.
     Defaults to `false`
+  * `emitClose` {boolean} Whether or not the stream should emit `close`
+    after it has been destroyed. Defaults to `true`
```

Comment:
This should be documented with a `changes:` YAML changelog entry for this heading (grep around for that string in the docs to see how it’s used)